### DISCUSSION TITLE
キーボール39　トラブル

### DISCUSSION THREAD

**Discussion by syabusyabu** [Yowkees/keyball#710] [Q&A]
キーボール39を購入し、組み立てやファームウェアの書き込みが終わり左側をPCに接続して使用していたら、突然右側のほうだけ反応しなくなり、その後右側をPCに接続しても赤く光るだけで認識されなくなってしまいました。
どうすればいいのでしょうか。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

左右の通信に不具合がある場合、TRSジャックの半田付け不良が原因の場合が多いです。
半田が基板の銀パッドに馴染んで、半田が山型になるように盛ってください。

解決しないようでしたら、半田付け面のお写真を見せていただければ幸いです。何かわかるかも知れません。

以上、お手数ですがご確認お願いいたします。

**Reply by syabusyabu** [reply]
返信が遅れてしまい申し訳ございません
左右の通信はケーブルを変えることで改善したのですが今度はトラックボールに不具合が出てしまいました
不具合の内容としては
・最初のうちは正常に動く
・数分後に突然トラックボールの数値が127になる
・数秒後に数値が0になり、入力を受け付けなくなる
・トラックボール基盤を刺し直すと最初のうちは正常に動くがまた壊れる
という物です
画像を添付しておきます
あとトラックボールホルダーとケースを止めるネジがなめってしまったので規格を教えていただけると幸いです![image](https://github.com/user-attachments/assets/c3e2e763-67b1-4429-97d6-e833bd702dd5)![image](https://github.com/user-attachments/assets/ab8817da-ad1f-4d7b-8f5b-b47077962efb)![image](https://github.com/user-attachments/assets/290a0e83-1f9b-4e36-95ee-66fba1307ecf)
